### ISSUE TITLE
Step 3 in guide for first-time users not working

### ISSUE THREAD

**Issue by chjo-hb** [kubeapps/kubeapps#752]
In step 3 it says to run some code, it does however not specify exactly where this particular code is supposed to be run. 
I tried running it in my PowerShell (windows 10), and got the following error message,

`The term 'export' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.`

Step 3 in readme says to run following code,
```
export POD_NAME=$(kubectl get pods -n kubeapps -l "app=kubeapps,release=kubeapps" -o jsonpath="{.items[0].metadata.name}")
echo "Visit http://127.0.0.1:8080 in your browser to access the Kubeapps Dashboard"
kubectl port-forward -n kubeapps $POD_NAME 8080:8080
```
I have managed to install KubeApps, and generated a token. But step 3 is very unclear and not working. I am a first time user with Kubernetes.

**Comment by chjo-hb** [comment]
I also tried changing to `set`. But then I get an issue with port forwarding after that.
`error: TYPE/NAME and list of ports are required for port-forward`

Using Windows 10 with PowerShell.

**Comment by chjo-hb** [comment]
Yes, I figured it out eventually. 
It might be a good idea to include these instructions in the main guide since Windows works a bit different from UNIX-systems and alike. 

In the end, I decided to stay on my current Vagrant + VirtualBox + Git-workflow since Minikube and Kubernetes require too much of my poor surface pro's hardware resources. But I will definitely keep an eye on it, seems to work well with GCP.

Thanx for your help! :)
